src/pages/druid-powered.md

```diff
@@ -8,9 +8,11 @@ title: Powered by Apache Druid
   </div>
 </div>
 
+
 :::tip
 
-<a href="https://github.com/apache/druid-website-src/blob/master/src/pages/druid-powered.md"><span class="fa fa-plus"></span> Add Your Company</a>
+<a href="https://github.com/apache/druid-website-src/blob/master/druid-powered.md">
```

Comment:
```suggestion
<a href="https://github.com/apache/druid-website-src/blob/master/src/pages/druid-powered.md">
```

Reply:
Looks like b/c of your additions, it's trying to override the URL fix Victoria made, so we can fix it manually here.